vagrant/scripts/config.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# To add changes in this file to a running VM, use command:
+# vagrant reload --provision 
+echo "*** Provisioning... ***"
+
+# Install dh and other useful tools:
+apt-get -y update
+#cat <<EOF > /etc/apt/apt.conf.d/local
+
+#Dpkg::Options {
+#   "--force-confdef";
+#   "--force-confold";
+#};
+
+#EOF
+
+#DEBIAN_FRONTEND=noninteractive apt-get -yq install openssh-server
+#apt-get -y upgrade
+#apt-get -y install devscripts debmake git cmake
+
+# Set locale using method from:
+# https://www.thomas-krenn.com/en/wiki/Perl_warning_Setting_locale_failed_in_Debian
+locale-gen en_GB.UTF-8
+localedef -i en_GB -f UTF-8 en_GB.UTF-8
+# NB another method (I don't recall where from) works if you do:
+# echo "export LC_ALL=C" >> .bashrc ; source ./.bashrc
+
+# Kaska set up
+wget https://repo.anaconda.com/archive/Anaconda3-2019.10-Linux-x86_64.sh
+sha256sum ./Anaconda3-2019.10-Linux-x86_64.sh
+
+# Run these manually (licence agreemewnt prompt issue)
+# bash ./Anaconda3-2019.10-Linux-x86_64.sh
+# source ~/.bashrc
+# source anaconda3/etc/profile.d/conda.sh
+# export PATH=${PATH}:anaconda3/bin
```

Comment:
We've got also this on the `.travis.yml` file. Also, if you run the `conda.sh` script, that should add the path properly. 
Carefule with adding the conda path at the end of your path will make that the python on this vm is not conda's python but the system one.